code/src/components/Album.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Icons from './Icons';
+
+const Album = (props) => {
+  const { albumsArray } = props
+  return (
+    <section className="album-container">
+      {albumsArray.map((mappedOutItem) => (
+        <div className="album-card" key={mappedOutItem.id}>
+          <div className="image-container">
+            <img src={mappedOutItem.images[0].url} className="album-image" alt="Cover" />
```

Comment:
The `img` element has an `alt` attribute, which is important for accessibility purposes.